jcl/src/java.base/share/classes/java/lang/invoke/GenerateJLIClassesHelper.java

```diff
@@ -49,4 +49,8 @@
 	static byte[] generateInvokersHolderClassBytes(String str, MethodType[] mts) {
 		throw OpenJDKCompileStub.OpenJDKCompileStubThrowError();
 	}
+
+        static byte[] generateInvokersHolderClassBytes(String str, MethodType[] invokerMts, MethodType[] callMts) {
```

Comment:
Is this method only required by `Java 11`? If so, pls decorate it with `JPP` flag accordingly.

Reply:
ie wrap the method with:
```
/*[INCLUDE-IF Java11]*/

/*[ENDIF]*/
```

Reply:
Done this now